solutions/remotemonitoring/armtemplates/basic.json

```diff
@@ -566,13 +552,6 @@
             "metadata": {
                 "description": "Azure Resource Manager url"
             }
-        },
-        "applicationInsightsInstrumentationKey": {
```

Comment:
I believe this values is provided by the SDS, are you sure we don't need this?

Reply:
It is needed but it is put into the other arm template keyvault.json instead of basic.json since it is going straight to keyvault rather than an environment variable in the VM.